themes/default/layouts/page/pricing.html

```diff
@@ -1635,8 +1635,8 @@ <h2 class="mb-12 text-center">Ready to try the Pulumi Service?</h2>
                                     <p class="text-xs pt-1 my-0">14-day trial. No credit card needed.</p>
                                 </div>
                                 <div class="flex flex-col justify-center align-center text-center w-full md:w-2/4 lg:mr-14 lg:ml-4">
-                                    <a href="{{ relref . "/contact" }}?form=sales" class="btn-secondary py-3 px-6">Contact Us</a>
-                                    <p class="text-xs pt-1 my-0">Contact us for a demo & quote.</p>
+                                    <a href="{{ relref . "/contact" }}?form=sales" class="btn-secondary py-3 px-6">Request a demo</a>
+                                    <p class="text-xs pt-1 my-0">Request a demo & quote.</p>
```

Comment:
This looks repetitive on the page

Perhaps it can say "Receive a demo & quote" so the CTA is the action of requesting and the bottom below is the result which is you receive.